ArtOfIllusion/src/artofillusion/RecentFiles.java

```diff
@@ -23,7 +23,7 @@ Changes copyright (C) 2017 by Maksim Khramov
 
 public class RecentFiles
 {
-  private static final int MAX_RECENT = 10;
+  public static final int MAX_RECENT = 10;
```

Comment:
This is making me just a bit twitchy...

Reply:
I understand! When I coded "recent scripts" persistency, I looked into the way AOI handled the "recent scenes" list, and I saw this constant. It seemes pointless to create another identical constant for the maximum length of "recent scripts" list, because actually these constants would achieve the same goal : describe the maximum number of items in a sub-menu listing files. So I made it public for reuse.
Ideally, the number of files in both the "recently used" scenes and scripts list should be a user preference, but I didn't want to extend the scope of this PR any further. What do you think about this reasoning? Should I actually code this "user pref" feature and add it to the PR?

Reply:
I'm a bit skeptical about making this configurable - it's really tied in with UI design. How do you display a long list? I really don't want to expose the constant as public, though.